include/test/mir_test_framework/async_server_runner.h

```diff
@@ -49,6 +49,9 @@ class AsyncServerRunner
 
     mir::Server server;
 
+protected:
+    std::string output_filename;
```

Comment:
If you're going to give derived classes direct access, at least make it const.

Reply:
Done.